tools/docs/quickstart.md

```diff
@@ -0,0 +1,39 @@
+# Quickstart of the Tools
+
+## Table of Contents
+
+1. [Running locally](#running-locally)
+   1. [Build the Tools](#build-the-tools)
+   1. [Run the Tools](#run-the-tools)
+
+## Running locally:
+
+- Tools subproject qualifier: `:tools`
+- Assuming your working directory is the repo root
+
+> **NOTE:** using the `-p` flag for `./gradlew` so not to specify the target subproject
+> on each task when running multiple tasks, but using the qualifier `:tools:` when
+> we have only one task to run.
```

Comment:
This is very unclear.  Perhaps the following is what was intended?
```suggestion
> **NOTE:** one may use the `-p` flag for `./gradlew` in order to avoid
> specifying the target subproject repeatedly on each task when running
> multiple tasks. When running only a single task, however, is is
> recommended to use the project qualifier (i.e. `:tools:`) for
> both simplicity and clarity.
```

Reply:
Yep, that is exactly what was meant. Thanks for the suggestion, makes it indeed more clear.

Reply:
✅ Done. Changed in other places respectively.